packages/block-editor/src/components/block-preview/index.js

```diff
@@ -20,11 +21,41 @@ import { BlockListItems } from '../block-list';
 
 export function BlockPreview( {
 	blocks,
-	__experimentalPadding = 0,
 	viewportWidth = 1200,
+	minHeight,
+	additionalStyles = [],
+	// Deprecated props:
 	__experimentalMinHeight,
-	__experimentalStyles = [],
+	__experimentalStyles,
+	__experimentalPadding,
 } ) {
+	if ( __experimentalMinHeight ) {
+		minHeight = __experimentalMinHeight;
+		deprecated( 'The __experimentalMinHeight prop', {
+			since: '6.2',
```

Comment:
Let's add a `version: '6.4'` and for `__experimentalPadding` below..

Reply:
I thought we weren't allowed to remove `__experimental` APIs that have made it into Core? Or is the discussion still ongoing? 😅

I'll add `6.4` nonetheless.